src/main/groovy/io/gatling/gradle/GatlingPlugin.groovy

```diff
@@ -80,12 +81,52 @@ class GatlingPlugin implements Plugin<Project> {
                 gatling "io.gatling.highcharts:gatling-charts-highcharts:${p.extensions.getByType(GatlingPluginExtension).toolVersion}"
 
                 if (gatlingExt.includeMainOutput) {
-                    gatlingImplementation project.sourceSets.main.output
+                    gatlingImplementation p.sourceSets.main.output
```

Comment:
As this closure grew up, perhaps renaming its parameter would remove a source of confusion.

Suggestion: `evaluatedProject`

Reply:
done